### PR TITLE
RuleSet - Check risky

### PR DESCRIPTION
**Pull request by SpacePossum**

### CODE REVIEW

File: tests/RuleSetTest.php [hunk 1]
```diff
@@ -228,6 +228,39 @@ public function providerSetDefinitionNames()
...
 
+    /**
+     * @param array $set
+     * @param bool  $safe
+     *
+     * @dataProvider provideSafeSets
+     */
+    public function testRiskyRulesInSet(array $set, $safe)
+    {
+        $this->assertSetSafety($set, $safe);
```
Comment (keradus) [inline tests/RuleSetTest.php]: now this test method is just a wrapper for private method. please remove that private one

**Comment by keradus** [comment]
Please rebase, solve raised issue and :+1:

File: src/Fixer/Alias/NoAliasFunctionsFixer.php [hunk 2]
```diff
@@ -93,8 +87,53 @@ public function fix(\SplFileInfo $file, Tokens $tokens)
...
+$a = show_source($filename, true);
+$a = sizeof($b);
+$a = strchr($haystack, $needle);
+'
+                ),
+            ),
+            null,
+            null,
+            null,
+            'Risky when the any of alias functions are overridden.'
```
Comment (SpacePossum) [inline src/Fixer/Alias/NoAliasFunctionsFixer.php]: note to self, grammar is not good

**Comment by keradus** [comment]
i let myself fix the testcase

**Comment by SpacePossum** [comment]
thanks for the help @keradus :)

**Comment by keradus** [comment]
Thank you @SpacePossum.

**Pull request closed without merging**